subsys/emul/i2c/emul_sbs_gauge.c

```diff
@@ -115,17 +115,21 @@ static int sbs_gauge_emul_transfer_i2c(const struct emul *target, struct i2c_msg
 					/* Return before writing bad value to message buffer */
 					return rc;
 				}
-				msgs->buf[0] = val;
+
+				sys_put_le16(val, msgs->buf);
```

Comment:
The endian conversion needs a comment. As is, this looks asymmetrical because the read path does the conversion and the write path doesn't.

Reply:
Done.